src/components/MyKiva/LendingStats.vue

```diff
@@ -16,6 +16,14 @@
 		</p>
 	</div>
 	<div
+		v-if="isNextStepsExp"
+	>
+		<div class="stats-wrapper tw-bg-white tw-rounded tw-shadow tw-p-1 md:tw-p-2 tw-w-full tw-flex tw-flex-col">
+			Goals Card goes here
```

Comment:
Goals card already exists on line 30, see this changeset:

https://github.com/kiva/ui/pull/6254